ipatests/test_integration/test_ipa_ipa_migration.py

```diff
@@ -426,22 +417,13 @@ def test_dry_run_record_output_ldif(self):
             extra_args=param,
         )
         assert self.replicas[0].transport.file_exists("/tmp/test.ldif")
+        assert result.returncode == 0
```

Comment:
The test is failing, please add an xfail annotation and report the issue upstream.